client/src/components/SignUpForm.js

```diff
@@ -1,77 +1,106 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
+import { withRouter } from 'react-router-dom';
+import axios from 'axios';
+import { useDispatch } from 'redux-react-hook';
+import * as actions from '../constants/actions';
+import useForm from '../hooks/useForm';
 
-class SignUpForm extends Component {
-  render() {
-    return (
-      <form action="/signup" method="POST">
-        <div className="form-group">
-          <label htmlFor="firstName" className="d-none">
```

Comment:
Sorry. Did we change the perameters for this? Let's make sure we don't change things unless mentioned. Let's add the first name (optional) back into the sign up form. Thank you.

Reply:
The form requirements should not have changed but somehow they did here as well..

<img width="684" alt="Screenshot 2019-07-09 13 06 50" src="https://user-images.githubusercontent.com/1711854/60908695-82f6c200-a24a-11e9-8400-9568d5bd6cdc.png">
<img width="639" alt="Screenshot 2019-07-09 13 07 02" src="https://user-images.githubusercontent.com/1711854/60908696-838f5880-a24a-11e9-9b18-b4fdb703f04c.png">

Reply:
You’re right. Added this back.